pages/threeView.js

```diff
@@ -48,15 +48,12 @@ window.ThreeView = function() {
         var spot1 = null;
 
         if (sceneInfo.addLights) {
-            var ambient = new THREE.AmbientLight(0x222222);
-            scene.add(ambient);
-
-            var directionalLight = new THREE.DirectionalLight(0xcccccc);
-            directionalLight.position.set(-1, 0, -2).normalize();
-            scene.add(directionalLight);
+          var hemispheric = new THREE.HemisphereLight(0xffffff, 0x222222, 1.2);
+          scene.add(hemispheric);
 
             if (sceneInfo.shadows) {
-                spot1 = new THREE.SpotLight(0xffffff, 1);
+                hemispheric.intensity = 0.5;
+                spot1 = new THREE.SpotLight(0xffffff, 0.7);
```

Comment:
That part of the code is not reachable in the viewer as of now.

My guess is that the viewer is based on [this example](https://threejs.org/examples/?q=gltf#webgl_loader_gltf_extensions), where shadows are toggled based on which model is viewed. I tweaked these settings, if shadows will be enabled in the future. 

If you want, I can either make a setting to enable shadows or expose an *enable-shadow checkbox* in the UI. But since the other viewers didn't have that feature, I didn't.